PennMobile/Dining/SwiftUI/DiningViewModelSwiftUI.swift

```diff
@@ -46,6 +46,12 @@ class DiningViewModelSwiftUI: ObservableObject {
         }
     }
 
+    func areAllVenuesEmpty(diningVenues: [VenueType: [DiningVenue]]) -> Bool {
```

Comment:
Why do we need to pass in diningVenues as a parameter? Could we just make this a computed property?